Lib/fontTools/varLib/models.py

```diff
@@ -280,7 +280,7 @@ def reorderMasters(self, master_list, mapping):
 		return new_list
 
 	def _computeMasterSupports(self):
-		supports = []
+		self.supports = []
```

Comment:
Although: doing many getattrs in the inner loop will be slower than appending to a local var and then setting the attr. Same for deltaWeights below.

Reply:
This may be true. This isn't hot code, however; for each table, the model is instantiated and the supports are computed once, then the submodels are used to generate deltas and interpolate.

Reply:
Both of those comments are correct indeed. I have mentally switched to always using a local. But am fine both ways.

Reply:
> This isn't hot code 

Fair enough.